spec/acceptance/reduce_worker_count_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper_acceptance'
```

Comment:
Perhaps `worker_count_spec.rb` is more apt? Since this tests increasing and decreasing it?

Reply:
the acceptance test only reduces it (which was the more difficult part to implement, relying on a custom fact to "remember" the previous worker count after the param has been changed)

I could add an acceptance test to increase it again if you see value in having that, either in this file or as a separate `increase_worker_count_spec.rb`. I suppose idempotence means either approach is sufficient, while increasing it again in the same test would be more computationally efficient

Reply:
I think I agree with @ehelms here and make it really a file about worker counts. Then you can also include the CPU-based tests in the file.

Reply:
I currently have the CPU based tests implemented in the unit tests but not in the acceptance tests.

I don't think we've used `override_facts` in acceptance tests before -- pretty neat if that works. (I think you are the original author of `override_facts`? so I assume you made this recommendation knowing that it works)

I will give this a try... thanks for the suggestions